analyzers/src/SonarAnalyzer.RuleDescriptorGenerator/SonarAnalyzer.RuleDescriptorGenerator.csproj

```diff
@@ -25,4 +25,9 @@
     <Copy SourceFiles="@(BinariesToCopy)" DestinationFolder="$(BinariesFolder)\RuleDescriptorGenerator" />
   </Target>
 
+  <ItemGroup>
```

Comment:
This should be after ItemGroup that has PackageRefrence, as they logically belong together